homework/git-homework-1.md

```diff
@@ -16,10 +16,19 @@
 ### Task 2
 1. Create a new repository on GitHub
     * It should either be public, or you must invite the teaching team as collaborators so we can see your work.
-1. Add your GitHub repository as a remote for your local `gitassignment` repository
+1. Add your GitHub repository as a remote for your local `githomework1` repository
     * `git remote add origin <url>`
-1. Push your local `gitassignment` repository to GitHub
+1. Push your local `githomework1` repository to GitHub
 1. Include your repository URL in your submission
 
+### Task 3
+1. Create a new branch called `playing-with-bash`
+2. Create 5 empty files with the names up to you
+3. Echo into the 5 empty files a few sentences about anything
+4. Stage, commit and push your work on GitHub
+5. Open up a pull request
```

Comment:
This should specify the destination (eg. own repo's main branch?)

Reply:
Usually when you open up a pull request, the destination branch where the changes will be merged is usually the `main` branch, so it can be safely assumed that is where the changes will be merged to at the end.

Reply:
In my experience, very little can be assumed for new learners, and it'd be easier to specify this directly in the homework.

Reply:
Good to know, I will change it later today